modules/dataflow_bucket/versions.tf

```diff
@@ -17,3 +17,11 @@
 terraform {
   required_version = ">= 0.12"
 }
+
+provider "google" {
```

Comment:
Providers should never be configured within a module. Instead, you can set version constraints on the https://www.terraform.io/docs/configuration/modules.html#provider-version-constraints-in-modules

Reply:
done